tasks/gulp/compile-assets.mjs

```diff
@@ -75,12 +78,16 @@ export function compileStylesheets () {
       gulp.src(`${slash(paths.fullPageExamples)}/**/styles.scss`, {
         sourcemaps: true
       })
-        .pipe(rename((path) => {
-          path.basename = path.dirname
-          path.dirname = 'full-page-examples'
-        }))
     )
+      .pipe(rename((path) => {
+        path.basename = path.dirname
+        path.dirname = 'full-page-examples'
```

Comment:
After Sass had compiled from `*.scss` to `*.css`, PostCSS was seeing the path as:

~`app/views/full-page-examples/campaign-page/styles.css`~
`app/views/full-page-examples/full-page-examples/campaign-page.css`

To reduce the chance of error in future let's rename last (before writing to disk)

1. Read source file to stream
2. Compile Sass from `*.scss` to `*.css`
3. Transform `*.css` using PostCSS
5. Rename or modify path
6. Write file to destination